webapp/packages/plugin-authentication/src/Dialog/AuthDialog.tsx

```diff
@@ -86,7 +88,12 @@ export const AuthDialog: DialogComponent<IAuthOptions, null> = observer(function
 
   async function login(linkUser: boolean, provider?: AuthProvider, configuration?: AuthProviderConfiguration) {
     try {
-      await dialogData.login(linkUser, provider, configuration);
+      const success = await dialogData.login(linkUser, provider, configuration);
+
+      if (!success) {
+        return;
+      }
```

Comment:
why not to throw en exception? now we have 3 states: success, false, exception according to try { if(success)